src/app/modules/project/overview/overview.component.ts

```diff
@@ -185,22 +185,18 @@ export class OverviewComponent implements OnInit {
     this.onInternalQueryChange();
 
     // Subscribe to search subject to debounce the input and afterwards searchAndFilter.
-    this.searchSubject
-      .pipe(
-        debounceTime(500)
-      )
-      .subscribe((result) => {
-        if (result == null) {
-          return;
-        }
-        this.onInternalQueryChange();
-      });
+    this.searchSubject.pipe(debounceTime(500)).subscribe((result) => {
```

Comment:
The previous way this was formatted is more clear in my opinion. It is also is in line with how subscriptions are formatted on line 314

Reply:
I definitely agree darn formatter is messing things up...